plugins/hello_world.py

```diff
@@ -98,7 +99,16 @@ def get(self):
             version=HelloWorld.instance.version,
             type=PluginType.simple,
             entry_point=EntryPoint(
-                href="./process/", ui_href="./ui/", data_input=[], data_output=[]
+                href="./process/",
```

Comment:
We should refactor these URLs *in another pull request* to rely on flasks url resolution from endpoint names.